examples/credential-registry/src/lib.rs

```diff
@@ -635,7 +597,6 @@ pub struct InitParams {
 /// It rejects if:
 ///   - Fails to log the events.
```

Comment:
- Fails to parse the input parameter.

This is an additional reason why the function invoke could reject.